Makefile

```diff
@@ -377,6 +378,20 @@ dist-cwd:
 	$(GZIP) $$tarball && \
 	rm -fr $$tardir
 
+vendored-dist: dist
+	$(Q)echo "Creating vendored dist tarball $(TAR_VERSION)..."; \
+	tardir=cri-resource-manager-$(TAR_VERSION) && \
+	tarball=cri-resource-manager-$(TAR_VERSION).tar && \
+	cp $$tarball$(GZEXT) vendored-$$tarball$(GZEXT) && \
+	$(GZIP_DC) vendored-$$tarball$(GZEXT) | tar -xf - && \
+	go mod vendor -v && \
+	mkdir -p $$tardir && \
+	  mv vendor $$tardir && \
+	$(TAR) -uf vendored-$$tarball $$tardir && \
+	rm -f vendored-$$tarball.* && \
+	$(GZIP) vendored-$$tarball && \
```

Comment:
why not to use directly tar compression parameters?

Reply:
`tar -uzf file.tar.gz` failed for me with `tar: Cannot update compressed archives`.

Reply:
Yes, like Antti said. tar can't update compressed archives.